src/styles/_catalog.scss

```diff
@@ -1,9 +1,26 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  margin: auto;
+  justify-content: center;
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
```

Comment:
you can create a variable for the card-size as you're using it a couple of times in your code